docs/usage/tutorials/reading_raster_data.ipynb

```diff
@@ -159,7 +159,7 @@
    "id": "83a6debf-e477-4b70-b82d-8e079ef8881e",
    "metadata": {},
    "source": [
-    "`RasterSource`s support `numpy`-like array slicing, so we can read a smaller chip within the full raster like so:"
```

Comment:
The 
```
`<class name>`s
```
convention is a deliberate choice. The trailing "s" is excluded from the inline code block to clarify that it is just for pluralization and not part of the class' name.

Reply:
It is inconsistent. Some places uses `s`, some not. 
https://docs.rastervision.io/en/0.21/usage/tutorials/reading_raster_data.html
![image](https://github.com/azavea/raster-vision/assets/5016453/89721056-f955-4cc1-994b-fd65c1a10ae3)

![image](https://github.com/azavea/raster-vision/assets/5016453/764bfcfd-6b48-45ed-bdd6-197ccaaa373a)

Reply:
That's true. The reason for that is a quirk of ReST/sphinx. The following is what I would like to do, but it does not work. 
```rst
:class:`.RasterSource`s
```